build.sbt

```diff
@@ -264,14 +266,6 @@ lazy val `scalafix-tests` = project
   .dependsOn(`scalafix-input`, `scalafix-rules`)
   .enablePlugins(BuildInfoPlugin, ScalafixTestkitPlugin)
 
-lazy val dontPublish = Seq(
-  publishArtifact := false,
-  packagedArtifacts := Map.empty,
-  publish := {},
-  publishLocal := {},
-  scalaModuleMimaPreviousVersion := None
```

Comment:
is there going to be an issue with this setting being missing where `don'tPublish` is now replaced by `publish / skip := true`?

Reply:
before, `scalaModuleMimaPreviousVersion` was set in `commonSettings` build-wide so then we had to replace that with `None`. why I did it that way originally I don't know — just not really thinking, I think?

but in any case, in the new version we only set it in the specific place where we actually want it (namely in `compat`'s `jvmSettings`), and just let it default to `None` everywhere else